entry_types/scrolled/package/src/contentElements/textBlock/TextBlock.module.css

```diff
@@ -1,7 +1,6 @@
 .text {
-  padding: 1em 0;
-  margin-top: 0;
-  margin-bottom: 0;
+  padding: 0;
+  margin: 1em 0;
   text-shadow: none;
```

Comment:
`.text` should have a min-height in order to keep empty text blocks selectable in editor - maybe gets added in separate text-selection PR?

Reply:
Can be part of a later PR. These style fixes are just about making selection rects render ok.